main.go

```diff
@@ -445,6 +480,122 @@ func deleteRun(_ *cobra.Command, args []string) {
 	fmt.Fprintf(stdout, "Successfully deleted secret if it existed: %s\n", object)
 }
 
+func isEditorAvailable(e string) (bool, error) {
```

Comment:
I think we should simplify this and use VISUAL/EDITOR directly. Then we don't have to check if the file exists (we can drop this function) and just throw an error in `findEditor`

Reply:
How about one final fallback to vi? (just like git)

Reply:
Eh. I think `git` is probably wrong here, especially in a world where Windows is so popular. I'd rather just fail with "Please set EDITOR" to be explicit.